Formula/kubecfg.rb

```diff
@@ -0,0 +1,48 @@
+class Kubecfg < Formula
+  desc "Manage complex enterprise Kubernetes environments as code"
+  homepage "https://github.com/ksonnet/kubecfg"
+  url "https://github.com/ksonnet/kubecfg/archive/v0.7.2.tar.gz"
```

Comment:
for making it easier to update you can use the github repo as `url` and the tag and commit as versions:

```
url "https://github.com/ksonnet/kubecfg.git"
      :tag => "v0.7.2",
      :revision => "7c1e6e59cc5cdf7259f17d349e55b6287b419b13"
```

Reply:
While that works fine, the Acceptable Formulae document [says](https://docs.brew.sh/Acceptable-Formulae#stable-versions):

> Tarballs are preferred to Git checkouts, and tarballs should include the version in the filename whenever possible.

Unless you specifically overrule that, I'm inclined to hew closely to it.